### DISCUSSION TITLE
APIMan on Mariadb database

### DISCUSSION THREAD

**Discussion by Satishshelake** [apiman/apiman#2666] [Q&A] [answered]
Hello,
  Is anybody tried to setup APIMan 3 on Mariadb 10.11 database? I have managed  to setup APIMan layover on widlfly 23 with mariadb. All wars are deployed but getting below exception even though all tables are created by liquibase change logs.

Caused by: org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: missing table [api_defs]

**Comment by volkflo** [comment, marked as answer]
Hello :wave: 

We are happily running apiman on postgresql without any flaws.
It is also the default driver that apiman ships in the bundle: https://github.com/apiman/apiman/issues/2265

Check out the docs: https://www.apiman.io/apiman-docs/installation-guide/dev/manager/database.html
It clearly says that mariadb 10 is not supported (yet).

The error you encounter is probably a missing entry in the liquibase schemas, you may get familiar with that and fix this issue and provide a PR. The are some good comments in the codebase on how to run the liquibase stuff.
The community would be very grateful if you provide a fix :pray: 

Additionally, if it is urgent, there is paid support and development for apiman.
